File: fix.py
import re

with open('d:/aa-enterprises/src/blogData.js', 'r', encoding='utf-8') as f:
    code = f.read()

if 'import { products }' not in code:
    code = "import { products } from './data';\n\n" + code

count = 0
def repl(match):
    global count
    p_index = count % 15
    count += 1
    return f"image: products[{p_index}]?.image || '/images/hero.png'"

code = re.sub(r"image:\s*'https://images\.unsplash\.com[^']+'", repl, code)

with open('d:/aa-enterprises/src/blogData.js', 'w', encoding='utf-8') as f:
    f.write(code)

print('Fixed images in blogData.js')

File: fix.js
const fs = require('fs');
let code = fs.readFileSync('d:/aa-enterprises/src/blogData.js', 'utf8');

if (!code.includes('import { products }')) {
  code = "import { products } from './data';\n\n" + code;
}

let count = 0;
code = code.replace(/image:\s*'(https:\/\/images\.unsplash\.com[^']+)'/g, (match, url) => {
  const pIndex = count % 15;
  count++;
  return \image: products[\]?.image || '/images/hero.png'\;
});

fs.writeFileSync('d:/aa-enterprises/src/blogData.js', code);
console.log('Fixed images in blogData.js');
